libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```diff
@@ -131,8 +131,10 @@ test-suite tests
     main-is:          Main.hs
     hs-source-dirs:   test
     other-modules:
+        Test.Cardano.Ledger.Binary
```

Comment:
This module is not needed

Reply:
you mean now that we are dropping `decode`?

Reply:
yep